### PR TITLE
Update and rename desktop-changelog.md to desktop-changelog.rst

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Comment by jasonblais** [comment]
In preparation for 3.5 desktop changelog PR

**Comment by esethna** [comment]
+1

**Comment by lfbrock** [comment]
@jasonblais it looks good, one question about the formatting here - was showing Team Name in italics intended? Are we doing that because "Team name" should actually be replaced by the team name? 

Might be something to discuss and decide on a guideline about. 

Also looks like shortcuts is misspelled. 

screenshot 2016-11-30 09 50 45

**Comment by jasonblais** [comment]
Updated formatting

File: source/help/apps/desktop-changelog.rst [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  Current team and channel name shown in window title bar
+-  Team tab is bolded for unread messages and has a red dot with a count of unread mentions
+-  Added an option to configure whether a red badge is shown on taskbar icon for unread messages
+-  Added new shortcuts:
+
+   -  CTRL+S: sets focus on the Mattermost search box
+   -  ALT+Left Arrow: go to previous page in history
```
Comment (jasonblais) [inline source/help/apps/desktop-changelog.rst]: Although I do have two questions to clarify:
 - should the keyboard buttons be bolded though? Or not?
 - should "Left Arrow" be in all capital letters? Given it's not the keyboard button name, like ENTER and CTRL, I think it shouldn't be

**Comment by lfbrock** [comment]
@jasonblais I think the placeholder text was going to be in square brackets or something other than italics as well?

**Comment by lfbrock** [comment]
+1

**Pull request merged**